.changeset/tough-moles-whisper.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-tech-radar': minor
+---
+
+Radar entry timeline added to the entry detail page
```

Comment:
Think that we should add some examples in here too so that people can see how to use this. I think there's also some examples in the readme that we could update too.

Reply:
The timeline object was already part of the datamodel from the beginning. Also the sample.ts this is being used. Only the UI part was not there yet.
I added a description of the data model to the readme.md, hope that will help people on how to use the timeline entries.